default/edit_list.conf

```diff
@@ -140,6 +140,9 @@ dkim_parameters			owner,privileged_owner 		hidden
 dkim_signature_apply_on		listmaster	 		write
 dkim_signature_apply_on		owner,privileged_owner 		hidden
 
+arc_parameters			listmaster			write
+arc_parameters			owner,privileged_owner		hidden
+
```

Comment:
arc_feature would be better to be added.

Reply:
Yes, added